### PR TITLE
Implement Web front end

### LINKED ISSUE #66: Make the Web part as functional as the mobile one
## Related feature
The objective is to have a web part functional for the MVP

## Improvement
The main page will be modified, by removing the template and adding the well-designed one.

## Screenshots (Optional)
See Figma from @JohanCDev

### PR DESCRIPTION
**Pull request by JohanCDev**
# New feature
We have a web front-end coherent with the app 
Capture d’écran 2023-02-06 à 08 39 57
Capture d’écran 2023-02-06 à 08 39 44
Capture d’écran 2023-02-06 à 08 39 54

Closes #66

### CODE REVIEW

**Comment by osvegn** [comment]
The pull request title can be improved

**Review by 31Nathan — DISMISSED** [review]
LGTM

File: Web/src/App.js [hunk 1]
```diff
@@ -1,20 +1,43 @@
...
 import { BrowserRouter, Routes, Route } from "react-router-dom"
+import { useState } from 'react';
 
 /**
  * @brief Main Function for AREA App
  * @returns
  */
 function App() {
+  const [userInformation, setUserInformation] = useState({
+    mail: "yoyoyo",
```
Comment (osvegn) [inline Web/src/App.js]: Seems invalid email

File: Web/src/App.js [hunk 2]
```diff
@@ -1,20 +1,43 @@
...
+    coord: {
+        latitude: 0,
+        longitude: 0,
+        city: ""
+    },
+    id: "testWallah",
```
Comment (osvegn) [inline Web/src/App.js]: Seems invalid ID

File: Web/src/AuthPage.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ export default function AuthPage() {
         await fetch("http://192.168.0.71:8080/" + endpoint, requestOptions).then(response => {
             response.json().then(data => {
                 console.log(data);
-                if (data.userUid != 'error') {
+                if (data.userUid !== 'error') {
```
Comment (osvegn) [inline Web/src/AuthPage.js:50]: Is it ok ?

**Review by osvegn — CHANGES_REQUESTED** [review]
Minor changes to do, and one question about code validity

File: Web/src/AuthPage.js (hunk 3 above)
Comment (JohanCDev) [reply Web/src/AuthPage.js:50]: Yes, it's checking if the values are the same AND if the type is the same. 
In JS, the condition 1 == '1' returns 'True'. 
While the condition 1 === '1' returns 'False'

**Review by osvegn — DISMISSED** [review]
Look good

**Review by 31Nathan — DISMISSED** [review]
LGTM

**Review by 31Nathan — APPROVED** [review]

**Review by osvegn — APPROVED** [review]
Look good

**Review by STom6 — COMMENTED** [review]
lgtm

**Pull request merged**